### PR TITLE
Documentation 3.10/add topics for graph creations

### PR DESCRIPTION
**Pull request by mchacki**
### Scope & Purpose

*DocuBlocks only PR. This will add code examples for HTTP create graph API*

### CODE REVIEW

File: Documentation/DocuBlocks/Rest/Graph/general_graph_create_http_examples.md [hunk 1]
```diff
@@ -206,7 +213,138 @@ A message created for this error.
 
   logJsonResponse(response);
 
-  graph._drop("myGraph", true);
+  graph._drop("smartGraph", true);
+@END_EXAMPLE_ARANGOSH_RUN
+
```
Comment (jsteemann) [inline Documentation/DocuBlocks/Rest/Graph/general_graph_create_http_examples.md:218]: Should there be an explanation for the example here?
All other examples have an explanation in front of them. This one hasn't.

File: Documentation/DocuBlocks/Rest/Graph/general_graph_create_http_examples.md [hunk 2]
```diff
@@ -178,14 +181,18 @@ A message created for this error.
   graph._drop("myGraph", true);
 @END_EXAMPLE_ARANGOSH_RUN
 
-@EXAMPLE_ARANGOSH_RUN{HttpGharialCreate2}
+Create a SmartGraph. This graph uses 9 shards and
+is sharded by the "region" attribute.
```
Comment (jsteemann) [inline Documentation/DocuBlocks/Rest/Graph/general_graph_create_http_examples.md:185]: Should it be mentioned here that the example will only work in the Enterprise Edition?

File: Documentation/DocuBlocks/Rest/Graph/general_graph_create_http_examples.md [hunk 3]
```diff
@@ -206,7 +213,138 @@ A message created for this error.
...
+
+assert(response.code === 202);
+
+logJsonResponse(response);
+
+graph._drop("smartGraph", true);
+@END_EXAMPLE_ARANGOSH_RUN
+
+Create an EnterpriseGraph. This graph uses 9 shards,
+it does not make use of specific sharding locations..
```
Comment (jsteemann) [inline Documentation/DocuBlocks/Rest/Graph/general_graph_create_http_examples.md:284]:
```suggestion
it does not make use of specific sharding attributes.
```

File: Documentation/DocuBlocks/Rest/Graph/general_graph_create_http_examples.md (hunk 1 above)
Comment (mchacki) [reply Documentation/DocuBlocks/Rest/Graph/general_graph_create_http_examples.md:218]: Good catch, there should be!

File: Documentation/DocuBlocks/Rest/Graph/general_graph_create_http_examples.md (hunk 2 above)
Comment (ansoboleva) [reply Documentation/DocuBlocks/Rest/Graph/general_graph_create_http_examples.md:185]: yes, please add "Available in the Enterprise Edition only."

File: Documentation/DocuBlocks/Rest/Graph/general_graph_create_http_examples.md [hunk 4]
```diff
@@ -206,7 +213,138 @@ A message created for this error.
...
+var response = logCurlRequest('POST', url, body);
+
+assert(response.code === 202);
+
+logJsonResponse(response);
+
+graph._drop("disjointSmartGraph", true);
+@END_EXAMPLE_ARANGOSH_RUN
+
+@EXAMPLE_ARANGOSH_RUN{HttpGharialCreateSmartWithSatellites}
```
Comment (ansoboleva) [inline Documentation/DocuBlocks/Rest/Graph/general_graph_create_http_examples.md:251]: this one's missing a brief intro as well.

**Review by ansoboleva — CHANGES_REQUESTED** [review]

**Comment by ansoboleva** [comment]
@mchacki thanks for adding the examples to 3.10. Once the comments are addressed, i'll run a job to generate the updated examples for the 3.10 version.

**Review by jsteemann — APPROVED** [review]
LGTM!

**Review by ansoboleva — APPROVED** [review]
lgtm

**Pull request merged**